A6question1.go

```diff
@@ -0,0 +1,71 @@
+//Sharanya Datrange
+package main
+import (
+	"fmt"
+)
+
+func main() {
+	MySlice := SliceInput()
+	accessSlice(MySlice)
+	fmt.Println("Testing Panic and Recover")
+}
+
+func SliceInput() []int {
+	var size int
+	fmt.Println("Enter the size of the slice:")
+	fmt.Scanln(&size)
+	MySlice := make([]int, size)
+	fmt.Printf("Enter %d elements for the slice:\n", size)
+	for i := 0; i < size; i++ {
+		fmt.Printf("Element %d: ", i)
+		fmt.Scanln(&MySlice[i])
+	}
+	return MySlice
+}
+
+func accessSlice(MySlice []int) {
+	//MySlice := [] int {2,4,6,8,10,12,14,16,18,20}
+	var index int
+	fmt.Println("Enter Input: ")
+	fmt.Scanln(&index)
+	found:=false
+	for i:=0;i<len(MySlice);i++ {
```

Comment:
what is need of for loop here? we just need to get item at the given index in slice if index is within limit else handle panic and print error.